cmds/boot/fitboot/main.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/boot/fit"
+)
+
+var (
+	dryRun    = flag.Bool("dryrun", true, "Do not actually kexec into the boot config")
+	debug     = flag.Bool("d", true, "Print debug output")
+	rootfs    = flag.String("r", "", "Root file system name")
+	cmdline   = flag.String("c", "earlyprintk=ttyS0,115200,keep console=ttyS0", "command line")
+	initramfs = flag.String("i", "", "InitRAMFS node name -- default none")
+)
+
+var v = func(string, ...interface{}) {}
+
+func main() {
```

Comment:
Couldn't this be part of kexec? Why a separate command?
The magic for fit is 0xd00dfeed.

Reply:
I'd rather it be separate. It's a really special case, currently used by only one user, and I hate to bloat kexec up with it.